src/components/core/tests/PivotTable.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import noop = require('lodash/noop');
+
+import { PivotTable, PivotTableInner, getGridDataSource } from '../PivotTable';
+import { oneMeasureDataSource } from '../../tests/mocks';
+import { pivotTableWithColumnAndRowAttributes } from '../../../../stories/test_data/fixtures';
+
+describe('PivotTable', () => {
+    it('should render PivotTableInner', () => {
+        const wrapper = mount(
+            <PivotTable
+                dataSource={oneMeasureDataSource}
+                getPage={noop as any}
+            />
+        );
+        expect(wrapper.find(PivotTableInner).length).toBe(1);
+    });
```

Comment:
Add tests like
- shows loading component until first returned page
- passes corect gridOptions to <AgGridReact>
- 
- maybe: export RowLoadingElement and test it directly